<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>629</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1358</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="editSearch"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonFind">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>Обзор</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="mybrowser" name="textBrowser"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editID">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Lesson:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editLesson"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editType">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Auditorium</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editAuditorium">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mybrowser</class>
   <extends>QTextBrowser</extends>
   <header>mybrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonFind</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>FindLesson()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>828</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>662</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>AddLesson()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>861</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>601</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>FindLesson()</slot>
  <slot>AddLesson()</slot>
 </slots>
</ui>
